cli-toolchain/graphql.md

```diff
@@ -634,6 +634,51 @@ You can use the *operations* argument to specify which operations are augmented
 - **update**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
 - **delete**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
 
+**Note**: When specifying operations as a part of the @auth rule, the operations not included in the list are not protected by default. For example, let's say you have the following schema:
+
+```graphql
+type Todo
+  @model
+  @auth(rules: [{ allow: owner, operations: [read] }])
+{
+  id: ID!
+  updatedAt: AWSDateTime! 
+  content: String!
+}
+```
+
+In this schema, only the owner of the object has the authorization to perform read (getTodo and listTodos) operations on the owner created object. But this does not prevent any other owner to update/delete some other owner's object. 
```

Comment:
What is "any other owner" or "other" in this statement/table? Can you add a "For instance, another owner may have access as they're part of...."